test/cypress/e2e/integration/app-frontend/validation.js

```diff
@@ -22,13 +22,12 @@ describe('Validation', () => {
       .blur()
       .focus()
       .clear()
-      .blur();
+      .blur()
     cy.get(appFrontend.fieldValidationError.replace('field', appFrontend.changeOfName.newFirstName.substring(1)))
       .should('exist')
       .should('be.visible')
-      .should('have.text', texts.requiredField)
-      .find(appFrontend.errorExclamation)
```

Comment:
I dont think `appFrontend.errorExclamation` is used anywhere else, so would be good to remove that definition as well